src/sparsezoo/model/model.py

```diff
@@ -326,7 +326,8 @@ def initialize_model_from_stub(
         if params:
             self._validate_params(params)
             self._stub_params.update(params)
-        path = os.path.join(CACHE_DIR, model_id)
+
+        path = os.path.join(SAVE_DIR, model_id)
```

Comment:
For unit testing this functionality, I think you'd have to move SAVE_DIR computation into this function so you could test different versions without a global variable controlling.

```
save_dir = os.getenv("...", CACHE_DIR)
```

then you could have a test like
```python
def test_env_var():
    os.environ["..."] = "mydir"
    _, path, _ = model.initialize_model_from_stub(...)
    assert path.startswith("mydir")

def test_no_env_var():
    _, path, _ = model.initialize_model_from_stub(...)
    assert path.startswith("~/.cache")
```